docs/docs/porting-an-html-site-to-gatsby.md

```diff
@@ -0,0 +1,543 @@
+
+export default () => (
+  {/* highlight-start */}
+  <Layout>
+    <h1>Welcome To Taylor's Tidy Trees</h1>
+    <h2>We care about trees of all kinds!</h2>
+  </Layout>
+  {/* highlight-end */}
+)
+```
+
+Have a check that the `Who We Are` link is still working.
+
+### Porting other pages
+
+Now it's time for the work to really pay off! Ella's page is a matter of using your `<Layout>` component again and copying in the main content. Don't forget to change `class` to `className`! A `breadcrumbs` prop can be passed to `<Layout>` to set the dynamic page title. Passing props is similar to an attribute on an HTML element:
+
+```jsx:title=/gatsby-site/src/pages/ella-arborist.js
+import React from "react"
+import Layout from "../components/Layout"
+import { Link } from "gatsby"
+
+export default () => (
+  {/* highlight-start */}
+  <Layout breadcrumbs={["Who We Are", "Ella"}}>
+  {/* highlight-end */}
+    <h1>Ella - Arborist</h1>
+    <h2>Ella is an excellent Arborist. We guarantee it.</h2>
+    <div className="bio-card">
+      <img
+        alt="Comically crude stick person sketch"
+        src="/person.png"
+      />
+      <p>Ella</p>
+    </div>
+  </Layout>
+)
+```
+
+The other 2 `Who We Are` pages for Marin and Sam can now be made with a similar structure. Maybe you are even thinking about another component for the Bio Card!
+
+Once `/services` and the root level HTML files are ported, here is what the finished Gatsby project file structure looks like:
+
+```
+gatsby-site
+├── static
+│   ├── favicon.ico
+│   └── person.png
+├── src
+│   ├── styles
+│   │   ├── normalize.css
+│   │   └── style.css
+│   ├── components
+│   │   └── Layout.js
+│   └── pages
+│       ├── index.js
+│       ├── about.js
+│       ├── contact.js
+│       ├── 404.js
+│       ├── who
+│       │   ├── index.js
+│       │   ├── ellla-arborist.js
+│       │   ├── marin-leafer.js
+│       │   └── sam-surgeon.js
+│       └── services
+│           ├── index.js
+│           ├── growing.js
+│           ├── cleaning.js
+│           └── shrinking.js
+├── node_modules
+├── .gitignore
+├── .prettierignore
+├── .prettierrc
+├── gatsby-config.js
+├── LICENSE
+├── package-lock.json
+├── package.json
+└── README.md
+```
+
+## Building & Deploying
+
+With your new Gatsby application taking shape, it's time to integrate it into your existing HTML website.
+
+### Gatsby build step
+
+You now have a site that mirrors the existing HTML site section. Stop the development server if it's still running; it's time to run the production build! 🎉
+
+```shell
+gatsby build
+```
+
+Once a build is complete, the compiled set of files can be found in `/public`.
+
+### Hosting the new website
+
+Once built, the contents of the `/public` folder are ready to be hosted at the root (`/`) of a domain. The files can be deployed in exactly the same way that your existing HTML site was deployed. For more deployment options, see the [Deploying and Hosting page](/docs/deploying-and-hosting/).
```

Comment:
I don't think we'll know exactly how their site was deployed, and dragging the public folder into FTP can be a bit of a pain. Perhaps we could make it more open-ended?
```suggestion
Once built, the contents of the `/public` folder are ready to be hosted at the root (`/`) of a domain. The files can be deployed in similar ways to how your existing HTML site may have been deployed. For more deployment options including automation with git and cloud services, see the [Deploying and Hosting page](/docs/deploying-and-hosting/).
```